src/streams/StreamIcon.js

```diff
@@ -15,12 +15,11 @@ type Props = $ReadOnly<{|
 |}>;
 
 export default function StreamIcon(props: Props): Node {
-  const { color, style, isPrivate, isMuted, isWebPublic, size } = props;
+  const { color, style, isPrivate, isWebPublic, size } = props;
 
   let Component = undefined;
-  if (isMuted) {
-    Component = IconMute;
-  } else if (isPrivate) {
+
+  if (isPrivate) {
```

Comment:
This component `StreamIcon` is used in several other places besides the Streams screen. So this change affects them too.

It should therefore go in its own commit, with a commit message that describes the changes it's making and why.

That commit can come before the commit that changes the sorting.